configuration/checkstyle-rules.xml

```diff
@@ -184,7 +184,10 @@
             how the code is composed, with only a quick visual inspection.
         -->
         <module name="AvoidStarImport" />
-        <module name="AvoidStaticImport" />
+        <module name="AvoidStaticImport">
+            <property name="excludes" value="com.amplifyframework.core.model.query.QueryOptions.*" />
+            <property name="excludes" value="com.amplifyframework.core.model.query.QueryPaginationInput.*" />
+        </module>
```

Comment:
This doesn't actually happen under the source tree, though, does it? (I couldn't spot it.) If not -- I'd say, remove these. The `src/test` has `checkstyle-suppressions.xml` rules which allow static imports in test code. (Mockito, for example, is designed for use with static imports.)

Reply:
That's correct, I forgot to remove them, thanks for catching it!